spec/jekyll_seo_tag_spec.rb

```diff
@@ -196,6 +196,60 @@
       end
     end
 
+    context "with image.path, image.height, and image.width" do
```

Comment:
If you're up for it, I feel like the template could see a 20% drop in LOC with refactoring. Then a separate refactoring of the tests.